ui/components/component-library/button-primary/button-primary.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { ButtonBase } from '../button-base';
+import { BUTTON_PRIMARY_SIZES } from './button-primary.constants';
+
+export const ButtonPrimary = ({
+  className,
+  danger,
+  size = BUTTON_PRIMARY_SIZES.MD,
+
+  ...props
+}) => {
+  return (
+    <ButtonBase
+      className={classnames(className, 'mm-button-primary', {
+        'mm-button-primary--type-danger': danger,
+      })}
+      size={size}
+      {...props}
+    />
+  );
+};
+
+ButtonPrimary.propTypes = {
+  /**
+   * An additional className to apply to the ButtonPrimary.
+   */
+  className: PropTypes.string,
+  /**
+   * Boolean to change button type to Danger when true
+   */
+  danger: PropTypes.bool,
+  /**
+   * The possible size values for ButtonPrimary: 'BUTTON_PRIMARY_SIZES.SM', 'BUTTON_PRIMARY_SIZES.MD', 'BUTTON_PRIMARY_SIZES.LG',
+   * Default value is 'BUTTON_PRIMARY_SIZES.MD'.
```

Comment:
Should we just reference the global constants if they are available? I know I'm referencing `TEXT_FIELD_TYPES.PASSWORD` in the TextFieldBase PR but that's because there are no global options.

```suggestion
   * The possible size values for ButtonPrimary: 'SIZES.SM', 'SIZES.MD', 'SIZES.LG',
   * Default value is 'SIZES.MD'.
```